notify/drivers/sfdc.py

```diff
@@ -0,0 +1,275 @@
+# Copyright 2016: Mirantis Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import json
+import logging
+from xml.dom import minidom
+
+import requests
+from requests.packages.urllib3 import exceptions as urllib_exc
+
+from notify import config
+from notify import driver
+
+requests.packages.urllib3.disable_warnings(urllib_exc.InsecureRequestWarning)
+
+
+LOG = logging.getLogger("sfdc")
+LOG.setLevel(config.get_config().get("logging", {}).get("level", "INFO"))
```

Comment:
It should not be configured in this way right in the code, it have to be done through [the logging configuration](https://docs.python.org/3/library/logging.config.html), but if you want to set default logging level you have to specify:

```
LOG.setLevel(logging.INFO)
````
and nothing more because the whole logging should be configured in one place.